cadence/contracts/FUSD.cdc

```diff
@@ -41,16 +65,61 @@ pub contract FUSD: FungibleToken {
     // out of thin air. A special Minter resource needs to be defined to mint
     // new tokens.
     //
-    pub resource Vault: FungibleToken.Provider, FungibleToken.Receiver, FungibleToken.Balance {
+    access(all) resource Vault: FungibleToken.Vault {
 
         // holds the balance of a users tokens
-        pub var balance: UFix64
+        access(all) var balance: UFix64
 
         // initialize the balance at resource creation time
         init(balance: UFix64) {
             self.balance = balance
         }
 
+        // TODO: REMOVE BELOW
+        // BEGIN OLD INTERFACE METHODS
+        access(all) view fun getBalance(): UFix64 {
```

Comment:
FYI these are included because the EVM CLI pre-release still has the older version of FT standards and requires these methods for interface conformance & successful deployment to emulator